rust_snuba/src/processors/replays.rs

```diff
@@ -357,6 +369,12 @@ struct ReplayEventLinkEvent {
     warning_id: Option<Uuid>,
 }
 
+#[derive(Debug, Deserialize)]
+struct ReplayViewedEvent {
+    sentry_user_id: u64,
```

Comment:
Makes sense, I'll change it. The idea was to explicitly mark this as a sentry user id. But it could be inferred from the u64 type and I guess the user id comes from the request context (not user provided).

Reply:
In this context, we don't know what this id is or where it comes from.  All we know is that the value of this field should be inserted into the column `viewed_by_id`.  The best name for that field is therefore `viewed_by_id`.  Think data first; systems second.  This is just a message.